_tests/apps_test.go

```diff
@@ -38,42 +38,40 @@ var _ = Describe("Apps", func() {
 		})
 
 		It("can create an app", func() {
-			sess, err := start("deis apps:create %s", app1Name)
-			Expect(err).To(BeNil())
-			Eventually(sess).Should(gexec.Exit(0))
-			Eventually(sess).Should(gbytes.Say("Creating Application... done, created %s", app1Name))
-			Eventually(sess).Should(gbytes.Say("Git remote deis added"))
-			Eventually(sess).Should(gbytes.Say("remote available at "))
-
-			sess, err = start("deis apps:destroy --confirm=%s", app1Name)
-			Expect(err).To(BeNil())
-			Eventually(sess).Should(gexec.Exit(0))
-			Eventually(sess).Should(gbytes.Say("Destroying %s...", app1Name))
-			Eventually(sess).Should(gbytes.Say("done in "))
-			Eventually(sess).Should(gbytes.Say("Git remote deis removed"))
+			output, err := execute("deis apps:create %s", app1Name)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(output).To(SatisfyAll(
+				ContainSubstring("created %s", app1Name),
+				ContainSubstring("Git remote deis added"),
+				ContainSubstring("remote available at ")))
+			output, err = execute("deis apps:destroy --confirm=%s", app1Name)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(output).To(SatisfyAll(
+				ContainSubstring("Destroying %s...", app1Name),
+				ContainSubstring("done in "),
+				ContainSubstring("Git remote deis removed")))
```

Comment:
Can you change this all back to the the `Eventually` logic? It's wise for us to move toward using `gexec`, as the error reporting is better and the control over each command is much better.